lib/dependencies/AMDDefineDependency.js

```diff
@@ -57,7 +57,7 @@ AMDDefineDependency.Template = class AMDDefineDependencyTemplate {
 			],
 			lf: [
 				"var XXX;",
-				"!(XXX = #.call(exports, __webpack_require__, exports, module))"
+				"!(XXX = #.call(exports, __webpack_require__, exports, module)); XXX === undefined && (XXX = module.exports)"
```

Comment:
Hey, thanks for the review! Can you please explain a bit about 
> Passing exports and module from the surrounding module is wrong here too.
I don't really get why that's wrong.